.github/workflows/ci.yml

```diff
@@ -52,11 +48,9 @@ jobs:
       - run: yarn test
 
   tests:
-    name: "Node ${{ matrix.node-version }} - ${{ matrix.os }} "
+    name: "Node ${{ matrix.node-version }} - ${{ matrix.os }}"
     runs-on: "${{ matrix.os }}-latest"
 
-    needs: [linting, basic-tests]
```

Comment:
These jobs have no output that's required for the `tests` job.
Do we really need `needs` here? Omitting it should speed up CI.